### ISSUE TITLE
HEVC QVBR mode doesn't follow set --qvbr-quality value

### ISSUE THREAD

**Issue by DimkaTsv** [rigaya/VCEEnc#87]
Can i ask question? Or report bug, if it is such. 
HQ-VBR, HQ-CBR and QVBR modes are actually also available for HEVC, not just AVC/AV1 as stated in documentation (as well as VBR-LAT, which isn't supported by VCEEncC, it seems?), and, while HQVBR and HQ-CBR get control via bitrate, as they basically use same flags, for HEVC QVBR specifically **--qvbr-quality** flag doesn't do anything, and encoder defaults to 23 (which is actual default for this parameter by AMF documentation), It does work for AVC QVBR though... And, probably, for AV1, but i cannot test it

Link to AMF documentation, proving that these 3 encoders are implemented for HEVC. VCEEncC also doesn't error out, so, there is that also.
https://github.com/GPUOpen-LibrariesAndSDKs/AMF/blob/master/amf/doc/AMF_Video_Encode_HEVC_API.md#221-static-properties

Used command 
,\VCEEncC64.exe -i "C:\Users\----\Videos\Radeon ReLive\Black Desert Online\Test_video.mp4" -o "Test_video_processed.mp4"  --codec hevc --profile main --level 5.2 --preset slow --ref 4 --ltr 3 --pa --qvbr 10000 **--qvbr-quality 51** --audio-copy

Number 51 is just for show, it doesn't  work for ANY value. And 51 should be possible to set anyways, right? 

```
C:\Users\----\Videos\Radeon ReLive\Black Desert Online>C:\----\----\Desktop\Utilites\VCE_Encoder\VCEEncC_8.12_x64\\VCEEncC64.exe -i "C:\Users\----\Videos\Radeon ReLive\Black Desert Online\Test_video.mp4" -o "Test_video_processed.mp4"  --codec hevc --profile main --level 5.2 --preset slow --ref 4 --ltr 3 --pa --qvbr 10000 --qvbr-quality 51 --audio-copy
--------------------------------------------------------------------------------
Test_video_processed.mp4
--------------------------------------------------------------------------------
Pre-analysis will be enabled for QVBR mode.
VCEEnc (x64) 8.12 (r1466) by rigaya, May  4 2023 07:34:56 (VC 1934/Win)
OS:            Windows 11 x64 (22621) [UTF-8]
CPU:           AMD Ryzen 7 5800X3D 8-Core Processor [4.45GHz] (8C/16T)
GPU:           AMD Radeon RX 6750 XT [31.0.14057.5006]
AMF:           Runtime 1.4.29 / SDK 1.4.29
Input Info:    avvce: H.265/HEVC, 1920x1080, 60/1 fps
Output:        H.265/HEVC  main @ Level 5.2 (main tier)
               1920x1080p 1:1 60.000fps (60/1fps)
               avwriter: hevc, aac => mp4
Quality:       slow
QVBR:           10000 kbps
QVBR level:    23
Max bitrate:   60000 kbps
QP:            Min: 0:0, Max: 51:51
VBV Bufsize:   60000 kbps
Bframes:       0 frames
Pre Analysis:  sc medium, ss high, activity y
               lookahead 0, caq medium, paq none, taq 0, motion-qual none, ltr off
Ref frames:    4 frames
LTR frames:    3 frames
Motion Est:    Q-pel
Slices:        1
GOP Len:       600 frames
Others:        skipframe:on deblock
```

**Comment by rigaya** [comment]
[VCEEnc 8.18](https://github.com/rigaya/VCEEnc/releases/tag/8.18) should have fixed this problem, now --qvbr-quality value will be reflected also for HEVC encoding.

**Comment by DimkaTsv** [comment]
Can confirm fix in VCEEncC 8.18
Issue closed
